dev/idm-am-common-user/idm/script/ui/notification/userNotifications.js

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
You probably don't need this either. I only added it for backwards compatibility with 6.0 end-user clients which want to be able to read notifications using the same API as 6.0.